Aquarius/Src/Aquarius/Sound/Devices.cpp

```diff
@@ -40,32 +50,21 @@ namespace Aquarius {
             }
         }
 
-        Device * Device::get()
-        {
-            static Device* device = new Device();
-            return device;
-        }
-
-        Device::~Device()
-        {
-            Deallocate();
-        }
```

Comment:
You might want this to still exist because idt the unique pointer destructor will handle this